app/assets/javascripts/models/scorecard.js

```diff
@@ -6,6 +6,7 @@ App.Scorecard = DS.Model.extend({
   user: DS.belongsTo('user', { async: true }),
   userInitials: DS.attr('string'),
   turns: DS.hasMany('turn', { async: true }),
+  isStarted: DS.attr('boolean'),
```

Comment:
I think I was hoping that updating a record would bring along a fresh copy of the data on its way back from the server, but it doesn't look like that's the case.